src/client/Microsoft.Identity.Client/Cache/Items/MsalAccessTokenCacheItem.cs

```diff
@@ -128,6 +137,7 @@ internal static MsalAccessTokenCacheItem FromJObject(JObject j)
                 ExpiresOnUnixTimestamp = expiresOn.ToString(CultureInfo.InvariantCulture),
                 ExtendedExpiresOnUnixTimestamp = extendedExpiresOn.ToString(CultureInfo.InvariantCulture),
                 UserAssertionHash = JsonUtils.ExtractExistingOrEmptyString(j, StorageJsonKeys.UserAssertionHash),
+                Kid = JsonUtils.ExtractExistingOrEmptyString(j, StorageJsonKeys.Kid)
```

Comment:
Will need to update the cache spec to include this new storage key as a reserved keyword.

Reply:
Yes, I've updated it already, it is part of the POP spec.